libs/api/domains/occupational-licenses/src/lib/models/occupationalLicense.model.ts

```diff
@@ -11,6 +11,14 @@ export enum OccupationalLicenseType {
   HEALTH = 'HEALTH_DIRECTORATE',
 }
 
+export enum HEATLH_DIRECTORATE_STATUS_TYPE {
+  valid = 'Í gildi',
+  limited = 'Í gildi - Takmörkun',
+  error = 'Ógilt',
```

Comment:
This will not be displayed to the user, those are the values that health directorate api status property. Only used to change UI colors and messages displayed to users.